eng/tools/suppressions/src/index.ts

```diff
@@ -0,0 +1,141 @@
+import { access, constants, readFile } from "fs/promises";
```

Comment:
I think it might be helpful to prove "file header comment" which is basically what you wrote in the PR:

```suggestion
/**
 * - Usage: `npx get-suppressions <tool-name> <path-to-file>`
 * - Returns: JSON array of suppressions, with specified tool name, applying to file (may be empty)
 * - Example: `npm get-suppressions TypeSpecRequirement specification/foo/data-plane/Foo/stable/2023-01-01/Foo.json`
 */
import { access, constants, readFile } from "fs/promises";
```

Reply:
Actually I think this is redundant with `getUsage()`, so I just moved this to the top of the file.